proposal/yenchanglu/README.rst

```diff
@@ -0,0 +1,81 @@
+=================
+Taking Your Order
+=================
+
+Basic information
+=================
+
+This is an automatically process to a virtual assistant that can take and process orders.
+
+You can use the following command to checkout from repository:
+
+``git clone https://github.com/yenchanglu/takingYourOrder``
+
+Problem to solve
+================
+
+It often feels like everything in our lives is designed to make the world move faster. An example of this modern impatience is the drive-thru: people became frustrated with waiting in slow drive-thru line. Why "fast food" restaurant speed of customer order processing is getting slow? Because it is still being done today as they were 50 years ago. This project is working on a way for modernizing the drive-thru, reducing unnecessary procedures and waiting time. It can also free up restaurant staff to focus on tasks that increase throughput. What's even better is that ordering food in a foreign language can be no longer frightening since the system will also translate the food and beverage menu in many different languages.
+
+Perspective users
+=================
+
+Clerk doesn't need to enter orders into POS and announce the total, receive payment and give change to customer anymore. Clerk just receives order and gives order to customer with this system. For hungry and thirsty people, it will make waiting in line less-annoying. It will also make travel easier for the foreign tourists.
+
+System architecture
+===================
+
+.. image:: architecture.png
+
+API description
+===============
+
+``welcome:: To set up an order and choose language``
```

Comment:
I would like to see the look and feel of the API, rather than the purposes alone.

Reply:
Yes, I've modified them. Thank you for your comment.